.github/workflows/docs.yaml

```diff
@@ -0,0 +1,23 @@
+name: Docs
+on:
+  push:
+    branches:
+      - "main"
```

Comment:
it probably doesnt matter so much for gh actions yaml config - but in Envoy where we have a lot of complex envoy configs we now use "least indent" for yaml formatting - my ~strong suggestion would be to adopt that as a policy as it makes the yaml more readable - esp where people use both for yaml lists

Reply:
Is there a way to configure yamllint to check for that?

Reply:
yep see here https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.indentation

specifically with `indent-sequences` set to `false`

Reply:
This is a great idea. The challenge is that there's a _ton_ of stuff in the Envoy Gateway project that fails lint when you do it. 😂 I'm going to hold off on this until we can discuss it.

Reply:
Resolving after switching to `indent-sequences: consistent`, per discussion with the group.